cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -43,21 +43,80 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
     // `appToggles` tells us whether a feature is toggled on or off in the current environment.
     if (appToggles.mediaPlayer.enabled) {
       describe('Media Player', () => {
-        it('should render a placeholder image', () => {
-          cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-            ({ body }) => {
-              // `video` blocks can also contain audio.
-              const media = getBlockData('video', body);
-              if (media) {
-                cy.get('div[class^="StyledVideoContainer"]').within(() => {
-                  cy.get('amp-img')
-                    .should('have.attr', 'src')
-                    .should('not.be.empty');
-                });
-              }
-            },
-          );
-        });
+        // This test is being temporarily throttled to the service 'news'.
+        if (service === 'news') {
+          it('should have a Media Player', () => {
+            cy.viewport(600, 1008);
```

Comment:
Yeah we need the view port in order for the test to be able to access the media player, without it the test only runs through the default orientation of the screen which at that point in test doesn't have a media player.

Reply:
Just to check I understand correctly, are we changing the viewport size to ensure that the player is inside the viewport, because (I guess) otherwise the iframe does not load on AMP?

If that's right, perhaps it would be more readable (and less brittle) to try something like `cy.get(someSelector).scrollTo('top')` (see https://docs.cypress.io/api/commands/scrollto.html#Position)? We could also add a comment to clarify why we need to scroll before testing. Lastly we should probably move this inside the condition that starts at L53 so that the scroll/resize only happens if the test is actually being run.

Reply:
In Ray's absence, I have picked this issue up and have implemented the pre-scroll using [scrollIntoView()](https://docs.cypress.io/api/commands/scrollintoview.html#Syntax). Please see changes.